llvm/docs/LangRef.rst

```diff
@@ -15384,6 +15384,228 @@ trapping or setting ``errno``.
 When specified with the fast-math-flag 'afn', the result may be approximated
 using a less accurate calculation.
 
+'``llvm.asin.*``' Intrinsic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Syntax:
+"""""""
+
+This is an overloaded intrinsic. You can use ``llvm.asin`` on any
+floating-point or vector of floating-point type. Not all targets support
+all types however.
+
+::
+
+      declare float     @llvm.asin.f32(float  %Val)
```

Comment:
Is `llvm.asin.f16` intentionally excluded here and everywhere else below?

Reply:
Yeah. f16 is a target specific intrinsic. So there won’t be any general documentation for it.